_sass/custom/_home.scss

```diff
@@ -58,6 +59,47 @@
 
 .home {
 
+    .row.bg-purple {
+        background: $primary-90;
+        padding: 0 1rem;
+    }
+
+    .row h2 {
+        margin: 1rem auto;
+    }
+
+    .programs .col {
+        height: 80px;
+        align-content: center;
+        background: #fff;
+        margin: 4px;
+        padding: 0;
+        border-radius: 12px;
+    }
+
+    .programs .col a {
+        display: block;
+        position: relative;
+        width: 100%;
+        height: 100%;
+        align-content: center;
+        border-radius: 12px;
+    }
+
+    .programs .col a::after {
+        content: "";
+        position: absolute;
+        top: 0;
+        left: 0;
+        width: 100%;
+        height: 100%;
+    }
+
+    .programs .col a:hover {
+        background-color: $primary-80;
```

Comment:
Could we break the purple with the other colour of your figma project?

Reply:
As discussed with Elena in Slack, we should vote on the secondary colour later and only use the primary tones for now :)